--- src/kcompat_std_defs.h	2021-09-22 19:13:19.000000000 +0200
+++ /usr/src/i40e-2.17.4/kcompat_std_defs.h	2021-11-10 13:47:11.000000000 +0100
@@ -91,6 +91,11 @@
 #define HAVE_TCF_EXTS_FOR_EACH_ACTION
 #endif /* 4,19,0 */
 
+/* HvB hack for debian 10 */
+#if (LINUX_VERSION_CODE < KERNEL_VERSION(4,19,195))
+#define NEED_SKB_FRAG_OFF_ACCESSORS
+#endif
+
 /*****************************************************************************/
 #if (LINUX_VERSION_CODE < KERNEL_VERSION(5,1,0))
 #else /* >= 5.1.0 */
@@ -118,7 +123,7 @@
 
 /*****************************************************************************/
 #if (LINUX_VERSION_CODE < KERNEL_VERSION(5,4,0))
-#define NEED_SKB_FRAG_OFF_ACCESSORS
+/*HvB #define NEED_SKB_FRAG_OFF_ACCESSORS*/
 #define NEED_FLOW_INDR_BLOCK_CB_REGISTER
 #else /* >= 5.4.0 */
 #define HAVE_XSK_UNALIGNED_CHUNK_PLACEMENT
