diff --git a/debian/i40e-dkms.dkms b/debian/i40e-dkms.dkms
index 9efd4bd..fad7163 100644
--- a/debian/i40e-dkms.dkms
+++ b/debian/i40e-dkms.dkms
@@ -2,5 +2,7 @@ PACKAGE_NAME="i40e"
 PACKAGE_VERSION="#MODULE_VERSION#"
 BUILT_MODULE_NAME[0]="$PACKAGE_NAME"
 DEST_MODULE_LOCATION[0]="/updates/dkms/intel/ethernet"
+BUILT_MODULE_NAME[1]="auxiliary"
+DEST_MODULE_LOCATION[1]="/updates/dkms/intel/ethernet"
 AUTOINSTALL="YES"
 REMAKE_INITRD="YES"
diff --git a/src/Makefile b/src/Makefile
index 62220eb..96e2149 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -10,6 +10,8 @@ ifneq ($(KERNELRELEASE),)
 ccflags-y += -I$(src)
 subdir-ccflags-y += -I$(src)
 
+NEED_AUX_BUS = 2
+
 obj-m += i40e.o
 
 i40e-y := i40e_main.o \
